Site/webapp/wdkCustomization/js/client/component-wrappers/Page.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { Props } from '@veupathdb/wdk-client/lib/Components/Layout/Page';
+
+// import { useAttemptActionClickHandler } from '@veupathdb/study-data-access/lib/data-restriction/dataRestrictionHooks';
+
+import UIThemeProvider from '@veupathdb/coreui/dist/components/theming/UIThemeProvider';
+import { colors } from '@veupathdb/coreui';
+import { useCoreUIFonts } from '@veupathdb/coreui/dist/hooks';
+
+export function Page(DefaultComponent: React.ComponentType<Props>) {
+  return function MicrobiomePage(props: Props) {
+    // useAttemptActionClickHandler();
+    useCoreUIFonts();
+
+    return (
+           <UIThemeProvider
+              theme={{
+                palette: {
+                  primary: { hue: colors.mutedBlue, level: 500 },
+                  secondary: { hue: colors.mutedRed, level: 500 },
+                },
+              }}
+            >
+              <DefaultComponent {...props} />
```

Comment:
You could put the snackbar-related components here, in between the `UIThemeProvider` and the `DefaultComponent`.